packages/scheduler-prisma/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@steveojs/scheduler-prisma",
-  "version": "6.0.1",
+  "version": "6.0.2-beta",
```

did you publish it to test locally?

we should move away from doing that and just using `yarn link`

---

Noted

---

```suggestion
  "version": "6.0.2",
```

@VLDCNDN this needs to drop the `-beta`